collect_app/src/main/java/org/odk/collect/android/backgroundwork/AutoSendTaskSpec.java

```diff
@@ -102,10 +103,10 @@
                     } catch (SubmitException e) {
                         switch (e.getType()) {
                             case GOOGLE_ACCOUNT_NOT_SET:
-                                notifier.onSubmission(true, Collect.getInstance().getString(R.string.google_set_account));
+                                notifier.onSubmission(true, TranslationHandler.getString(Collect.getInstance(), R.string.google_set_account));
```

Comment:
We can actually use the `context` param here instead of `Collect.getInstance()`

Reply:
Done.